src/oauthClients.json

```diff
@@ -0,0 +1,5 @@
+[{
```

Comment:
Just hard-code it. I don't think there will be other clients for a while.

I'm not even really sure how this is actually used :/